dashboard/test/controllers/hint_view_requests_controller_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+class HintViewRequestsControllerTest < ActionController::TestCase
+
+  setup do
+    @student = create :student
+  end
+
+  test 'creation requires current_user' do
+    HintViewRequest::ENABLED = true
+    post :create, {}, format: :json
+    assert_response :unauthorized
+  end
+
+  test 'creation requires params' do
```

Comment:
nit: maybe split up into separate "creation fails without params" and "can be created"